### PR TITLE
Mes-6271: unsubmittied test banner

### PR DESCRIPTION
**Pull request by andrewsetterfield**
## Description

Migrate from des3

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional
Screenshot 2021-10-27 at 15 14 16
)

### CODE REVIEW

**Review by solomonsimukoko — APPROVED** [review]

File: src/app/pages/dashboard/dashboard.page.ts [hunk 1]
```diff
@@ -38,6 +39,9 @@ interface DashboardPageState {
 })
 export class DashboardPage extends BasePageComponent {
 
+  @ViewChild(IncompleteTestsBanner)
+  incompleteTestsBanner: IncompleteTestsBanner;
+
```
Comment (matthew2564) [inline src/app/pages/dashboard/dashboard.page.ts]: This is unused in DES3, assume can be removed from here also

File: src/app/pages/journal/journal.page.ts [hunk 2]
```diff
@@ -69,9 +68,8 @@ export class JournalPage extends BasePageComponent implements OnInit {
   @ViewChild('slotContainer', { read: ViewContainerRef }) slotContainer;
   @ViewChild(IonContent) content: IonContent;
 
-  // @TODO - Reintroduce with MES-6271
-  // @ViewChild(IncompleteTestsBanner)
-  // incompleteTestsBanner: IncompleteTestsBanner;
+  @ViewChild(IncompleteTestsBanner)
+  incompleteTestsBanner: IncompleteTestsBanner;
```
Comment (matthew2564) [inline src/app/pages/journal/journal.page.ts]: This is also unused in DES3, please remove

File: src/app/pages/dashboard/dashboard.page.ts (hunk 1 above)
Comment (andrewsetterfield) [reply src/app/pages/dashboard/dashboard.page.ts]: removed, didnt effects screens.

File: src/app/pages/journal/journal.page.ts (hunk 2 above)
Comment (andrewsetterfield) [reply src/app/pages/journal/journal.page.ts]: removed, didnt effects screens.

**Review by matthew2564 — APPROVED** [review]

**Review by solomonsimukoko — APPROVED** [review]

**Pull request merged**